clang/bindings/python/tests/cindex/INPUTS/a.inc

```diff
@@ -0,0 +1 @@
+1, 2, 3
```

Comment:
Lack of newline at the end of the file doesn't seem to be an important part of the test. You should add it to both `a.inc` and `b.inc`